spec/features/visitor_can_decrease_item_from_cart_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+ describe "Visitor cart has an item in it" do
+   describe "visitor sees that her cart has a quantity of 2" do
+     describe "the visitor can remove the item" do
+       it "visitor sees that she has 1 items in her cart and the subtotal of her order decreases" do
+          create_items
+          visit 'items_path'
+
+          click_on "Add to cart"
+          click_on "Add to cart"
+
+          visit "/cart"
+          expect(page).to have_content(2)
+          expect(page).to have_content(39.98)
+          expect(page).to have_content(@item.title)
+          click_on "decrease quantity"
```

Comment:
We'll want to be careful using the `decrease quantity` here as we'll be changing the link to a `-` button.